### DISCUSSION TITLE
Vulkanised 2024

### DISCUSSION THREAD

**Discussion by theodoregoetz** [vsg-dev/VulkanSceneGraph#1008] [General]
I'm curious to know if any VSG developers or users might show up at Vulkanised 2024.

[Vulkanised 2024](https://vulkan.org/events/vulkanised-2024)
The 6th Vulkan Conference | Mountain View, CA. USA | Feb 5-7, 2024

**Comment by timoore** [comment]
Do you know if it coincides with any other graphics conference or meeting?

**Reply by theodoregoetz** [reply]
I suppose [EI (Electronic Imaging)](https://www.imaging.org/IST/Conferences/EI/EI2024) is near in both space and time (San Francisco, California, January 21-25, 2024) and they do have a section dedicated to "Visualization and Data Analysis" where VSG would fit nicely I think, but I don't see much else until March with [GDC](https://gdconf.com/).
